### PR TITLE
Updates the regression notebook

### PR DESCRIPTION
**Pull request by jwdebelius**
Notebook includes bloom counts

### CODE REVIEW

File: ipynb/Alpha Diversity Regression.ipynb [hunk 1]
```diff
@@ -83,28 +91,59 @@
...
-   "execution_count": 13,
+   "execution_count": 47,
    "metadata": {
     "collapsed": false
    },
    "outputs": [],
    "source": [
     "def filter_map(x):\n",
-    "    country = x['country'] in {'USA', 'United Kingdom', 'Canada'}\n",
+    "    country = x['country'] in {'USA', 'United Kingdom', 'Canada', 'Australia'}\n",
```
Comment (wasade) [inline ipynb/Alpha Diversity Regression.ipynb]: The comment above says only US / UK?

File: ipynb/Alpha Diversity Regression.ipynb [hunk 2]
```diff
@@ -83,28 +91,59 @@
...
     "    bmi = (x['bmi_corrected'] < 40) and (pd.isnull(x['bmi_corrected']) == False)\n",
     "    sex = x['sex'] in {'male', 'female'}\n",
     "    poo = x['bowel_movement_quality'] not in {\"I don't know, I do not have a point of reference\"}\n",
     "    race = x['race'] in {'Caucasian', 'Asian or Pacific Islander', 'Other'}\n",
-    "    cpn = x['center_project_name'] in {'AG1', 'AG11', 'AG13', 'AG14', 'AG17', 'AG18', \n",
-    "                                       'AG19', 'AG2', 'AG20', 'AG3', 'AG4', 'AG5', \n",
-    "                                       'AG6', 'AG7', 'AG9', 'AGFermentation', \n",
-    "                                       'AGUNDX', 'AGagOffice1', 'AGlost_samples'}\n",
-    "    return country & bmi & sex & poo & race & cpn"
+    "    cpn = x['center_project_name'] in {'AG31', 'AGag35', 'AG27', 'AGag33', 'AGag36', \n",
```
Comment (wasade) [inline ipynb/Alpha Diversity Regression.ipynb]: Why is there a limiting to specific rounds and why'd that change?

Comment (jwdebelius) [reply ipynb/Alpha Diversity Regression.ipynb]: rounds are limited based on the sample size after filtering. So, if there are only 3 people in a run prior to filtering, it gets excluded.

**Comment by wasade** [comment]
Okay, thanks. At a minimum, could that be described in the notebook?
Ideally this is just done on the fly instead of hard coding

**Pull request merged**